pkg/report/output/html/security.tmpl

```diff
@@ -0,0 +1,31 @@
+	<div id="result-summary">
```

Comment:
Hello, Is that normal the summary inside the HTML report doesn't show warnings count ?

Reply:
Hi @joevin-slq-docto yes, currently the HTML report does not show warnings. However, we do plan to change this in the future, so that the HTML report does include warnings: https://github.com/Bearer/bearer/issues/1136